### ISSUE TITLE
Tests don't run/fail with mocha-headless-chrome while they succeed in browser

### ISSUE THREAD

**Issue by robhicks** [direct-adv-interfaces/mocha-headless-chrome#7]
We have started a lot of unit tests for ES6 modules. Our tests run fine in the browser but fail when using mocha-headless-chrome.

In some cases, mocha-headless-chrome attempts to run the tests but exists reporting 0 passing tests. Other times, it runs but all the tests fail because the web components apparently never get created. 

[Here's a repo](https://github.com/robhicks/mocha-tests) that shows the first case. Any help would be greatly appreciatd.

**Comment by dima117** [comment]
Hi,

I made the actions which you listed in the [readme](https://github.com/robhicks/mocha-tests/blob/master/README.md) and I see in the terminal the same result as in the browser.

![](https://habrastorage.org/webt/pm/q6/ij/pmq6ijoodsxlnsooyz78yyhr0l4.png)

![](https://habrastorage.org/webt/ax/v4/46/axv4465_rlxp_dx1bzvrepbyyes.png)

Please give me more info how to reproduce the problem.

**Comment by dima117** [comment]
Hi,

I am fixed the bug when [final handler](https://github.com/direct-adv-interfaces/mocha-headless-chrome/commit/c1c3abc81c3e9a44b84c96ec44b332283b01fa5e#diff-b9c2b69f1fd91e39237ff14152db7467R65) wasn't call in some cases. Perhaps it can fix your problem. Please update `mocha-headless-chrome` to `1.7.1` version.

**Comment by robhicks** [comment]
Thanks! I'll take a look and also investigate why the test repo that I put together for this issue worked differently for you.

**Comment by dima117** [comment]
Hi,

I'm sorry for the very late reply.

I cloned your repo to know about the problem in more detail. And I found that the path to the tested file was incorrectly specified in the `npm test` command. The path to the local file `index.html` was specified, but the web server address `http://localhost:8080` must be specified. This is necessary for the tools that you use in the tests.

I sent you the [PR](https://github.com/robhicks/mocha-tests/pull/1) which fix the path.
